recipe/conda_build_config.yaml

```diff
@@ -9,20 +9,23 @@ gcc_version:
   - 8.5.0
   - 9.5.0
   - 10.4.0
```

Comment:
Can you remove these? There's no point keeping them since they don't receive security updates.

Reply:
I mean it's possible that we'd see a 10.5 I guess, but happy to remove them. 🤷